buku

```diff
@@ -499,6 +530,8 @@ class BukuDb:
         if tag_manual is not None:
             if append_tag:
                 self.append_tag_at_index(index, tag_manual)
+            elif delete_tag:
```

Comment:
Please add the check for `delete_tag` flag in line 561 (similar to the current `append_tag` check) to make sure we don't end up refreshing all titles.